packages/modules/imodel-browser/src/containers/iModelTiles/IModelTile.tsx

```diff
@@ -68,6 +72,7 @@ export const IModelTile = ({
             apiOverrides={thumbnailApiOverride}
           />
         }
+        className="iac-tile"
```

Comment:
why are we creating new scss files for some components, and in others we are just applying styles via props directly?

Reply:
I like to use stylesheets instead of inline styles but since multiple developers work on this repo and some use inline styles and some use new files. so its a mixture throughout the repo. Maybe someone should clean it up and add everything through scss files instead of using inline styles